dev/demo_web/views/page_view.ex

```diff
@@ -9,6 +9,7 @@ defmodule DemoWeb.PageView do
     <h2>Links</h2>
     <ul>
       <li><%= link "Profile IndexController, :hello", to: Routes.page_path(DemoWeb.Endpoint, :hello) %></li>
+      <li><%= link "Profile IndexController, :hello with param", to: Routes.page_path(DemoWeb.Endpoint, :hello, name: "dev") %></li>
```

Comment:
Feel free to remove this line. It's just because I always use that link for tests.

Reply:
it's a good addition! :)